src/wire/tcp.rs

```diff
@@ -624,6 +625,7 @@ pub enum TcpOption<'a> {
     WindowScale(u8),
     SackPermitted,
     SackRange([Option<(u32, u32)>; 3]),
+    TimeStamp(u32, u32),
```

Comment:
This would be improved with named fields, though given the manipulation of this struct above (e.g. `reply_repr.timestamp = match (repr.timestamp, self.tsval_generator) {`) I think it would be even better as a proper structure with an inherent method to build a response struct.